frontend/src/component/changeRequest/ChangeRequestOverview/ApplyButton/ApplyButton.tsx

```diff
@@ -9,7 +9,8 @@ export const ApplyButton: FC<{
     disabled: boolean;
     onSchedule: () => void;
     onApply: () => void;
-}> = ({ disabled, onSchedule, onApply, children }) => (
+    variant?: 'create' | 'update';
+}> = ({ disabled, onSchedule, onApply, variant = 'create', children }) => (
```

Comment:
Hmm. I wonder if this should be handled outside the component? Or maybe a different button entirely? Or maybe the component should be renamed 🤔 Regardless, that can be done in a follow-up PR, so it's not a blocking thing.

But maybe `variant` should be a required prop instead?